tests/cromwell_config/docker.conf

```diff
@@ -0,0 +1,36 @@
+include required(classpath("application"))
```

Comment:
Ehm. Yes. Sorry, I am a bit late on this, but maybe we could also consider running pytest with sudo rights on travis-ci instead of having to add this conf file..
Does that fix the problem?

Reply:
No, the default docker configuration doesn't mount anything other than the cromwell execution folder. The output directory won't be accessible.

Reply:
:+1: